server/balancer.go

```diff
@@ -82,98 +85,158 @@ func (s *storageBalancer) GetResourceKind() ResourceKind {
 }
 
 func (s *storageBalancer) Schedule(cluster *clusterInfo) Operator {
-	region, source, target := scheduleStorage(cluster, s.opt, s.selector)
+	// Select a peer from the store with largest storage ratio.
+	region, oldPeer := scheduleRemovePeer(cluster, s.selector)
 	if region == nil {
 		return nil
 	}
 
-	diff := source.storageRatio() - target.storageRatio()
-	if diff < s.opt.GetMinBalanceDiffRatio() {
+	// We don't schedule region with abnormal number of replicas.
+	if len(region.GetPeers()) != s.rep.GetMaxReplicas() {
```

Comment:
We can, but we don't handle it in the storage balancer, because the replica checker will do that.